src/Tasks.UnitTests/RoslynCodeTaskFactory_Tests.cs

```diff
@@ -1190,18 +766,10 @@ private void TryLoadTaskBodyAndExpectFailure(string taskBody, string expectedErr
                 taskInfo.CodeType.ShouldBe(expectedCodeType.Value);
             }
 
-            if (expectedSourceCode != null)
+            if (verifySource)
             {
-                NormalizeRuntime(taskInfo.SourceCode)
-                    .ShouldBe(NormalizeRuntime(expectedSourceCode), StringCompareShould.IgnoreLineEndings);
+                Verify(taskInfo.SourceCode, _verifySettings).GetAwaiter().GetResult();
```

Comment:
Can you explain for the unenlightened how this is figuring out what it should be? I tend to trust hardcoded 'expected' values, whereas I'm a bit more finnicky about autogenerating them.

Reply:
Oh, the files are below. Well, I'm not sure if I like it more or less, but that at least makes me feel more comfortable, so 👍

Reply:
Yeah, it uses [`Verify`](https://github.com/VerifyTests/Verify) which we started using for similar snapshot testing for `-terminallogger` but I like it.